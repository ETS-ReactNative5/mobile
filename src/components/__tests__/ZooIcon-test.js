import 'react-native'
import React from 'react'
import renderer from 'react-test-renderer'
import ZooIcon from '../ZooIcon'

it('renders correctly', () => {
  const tree = renderer.create(
    <ZooIcon iconName={'nature'} />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
