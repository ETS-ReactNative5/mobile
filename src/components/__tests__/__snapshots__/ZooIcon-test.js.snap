// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  allowFontScaling={false}
  style={
    Array [
      Object {
        "color": undefined,
        "fontSize": 12,
      },
      Array [
        undefined,
        undefined,
      ],
      Object {
        "fontFamily": "zoo-font",
        "fontStyle": "normal",
        "fontWeight": "normal",
      },
      Object {},
    ]
  }
>
  ï„«
</Text>
`;
